phpBB/phpbb/install/module/install_finish/task/install_extensions.php

```diff
@@ -118,6 +118,13 @@ public function run()
 
 			try
 			{
+				$extension = $this->extension_manager->get_extension($ext_name);
+
+				if (!$extension->is_enableable())
+				{
+					continue;
```

Comment:
You should log it I think ($this->iohandler->add_log_message(array('CLI_EXTENSION_ENABLE_FAILURE', $ext_name));)

Reply:
Yes